introduction.md

```diff
@@ -1,3 +1,9 @@
 Hello My name is Rebecca. My favorite Code School Path is Javascript.
 
 I am learning about GitHub to become the go to person for Git at my company.  I am also going to convert the project I am working in from SVN to Git
+
+Deadly Skills
+* Git
+* Java
+* Javascript
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?